python/py-gitfs/Portfile

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                py-gitfs
+github.setup        presslabs ${python.rootname} 0.5.2
+revision            0
+
+categories-append   devel
+license             Apache-2
+maintainers         nomaintainer
+platforms           darwin
```

Comment:
please check whether any code gets compiled (a cursory look suggest it isn't); if that's the case please add `supported_archs noarch`